demo/test.html

```diff
@@ -137,6 +156,46 @@ <h3>Events</h3>
         });
       });
 
+      $("#trip-start-btn").click(function(e) {
```

Comment:
hot take but i miss jquery

Reply:
sometimes.. for quick 1 pager stuff yeah